ssp

```diff
@@ -2045,16 +2045,18 @@ sub print_ea3_php_configuration {
         }
 
         if ( $EA3_PHP4HANDLER && $EA3_PHP4HANDLER ne 'none' ) {
-            my @php_v = split /\n/, timed_run( 0, '/usr/local/php4/bin/php', '-v' );
-            if ( @php_v && $php_v[0] =~ /^PHP\s(\S+)\s(\S+)/ ) {
-                $EA3_PHP4VERSION = $1;
-            }
-            else {
-                $EA3_PHP4VERSION = '(version unknown)';
-            }
+            if ( -x '/usr/local/php4/bin/php' ) {
```

Comment:
This is not a correct fix.  timed_run isn't supposed to be able to return undefined output, so the "Use of uninitialized value" that triggered this fix should not have occurred.  Also, just because something is executable isn't going to guarantee that it would prevent uninitialized variable warnings.  There is more to the story that is not known at this time.

Reply:
> timed_run isn't supposed to be able to return undefined output

Thanks for catching that. I was not aware timed_run shouldn't behave this way. I'll see if I can find out more

> Also, just because something is executable isn't going to guarantee that it would prevent uninitialized variable warnings

You're absolutely right, let me see what I can do